src/rgw/rgw_rados.cc

```diff
@@ -3974,7 +3974,6 @@ int RGWRados::init_watch()
 
   librados::Rados *rad = &rados[0];
   int r = rad->ioctx_create(control_pool, control_pool_ctx);
```

Comment:
@theanalyst  If it returns a negative number (not -ENOENT)  here, we will get it.